sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -771,12 +785,15 @@ public Key restoreKey(byte[] backup) {
      * call {@link KeyClient#getKey(KeyBase baseKey)} . This will return the {@link Key keys} with key material included of the specified versions.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listKeyVersions}
      *
+     * <p><strong>Samples to iterate KeyBase by each page</strong></p>
```

Comment:
The sample description needs more description as to what its doing like the one above it, +  its doing it per page.